CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+## 15.0.1 (October 20, 2020)
+- Added several optimizations to improve performance for a variety of high-load simultaneous job launch use cases https://github.com/ansible/awx/pull/8403
+- awx.awx collection modules now provide a clearer error message for incompatible versions of awxkit - https://github.com/ansible/awx/issues/8127
+- Fixed a bug in notification bodies that contain certain unicode characters - https://github.com/ansible/awx/issues/7400
+- Fixed a bug that prevents the deletion of Workflow Approval records - https://github.com/ansible/awx/issues/8305
+- Fixed a bug that broke the selection of webhook credentials - https://github.com/ansible/awx/issues/7892
+- Fixed a bug which can cause confusing behavior for social auth logins across distinct browser tabs - https://github.com/ansible/awx/issues/8154
+- Fixed a bug in the output of Workflow Job Templates using the `awx export` tool - https://github.com/ansible/awx/issues/7798
+- Fixed a race condition that can lead to missing hosts when running parallel inventory syncs - https://github.com/ansible/awx/issues/5571
+- Fixed an HTTP 500 error when certain LDAP group parameters aren't properly set - https://github.com/ansible/awx/issues/7622
+- Updated a few dependencies in response to several CVEs:
+    . CVE-2020-7720
```

Comment:
This list renders weirdly in the markdown. View with:

<img width="255" alt="image" src="https://user-images.githubusercontent.com/773186/96497553-dd861880-1218-11eb-9482-8c12e3a592d7.png">

Looks like:

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/773186/96497577-e676ea00-1218-11eb-99a4-6c14279fc4ce.png">

Reply:
Fixed!